scripts/src/tests/services/index.spec.ts

```diff
@@ -31,6 +32,19 @@ describe("general api checks", async () => {
 			.expect(404);
 	});
 
+	test("app blockchain version should be 2 | 3", async () => {
+		const application = await helpers.createApp(generateId(IdPrefix.App));
+		const blockchainVersion = await getAppBlockchainVersionService(application.id);
+		expect(blockchainVersion === "2" || blockchainVersion === "3");
```

Comment:
It would be better to check the specific blockchain version

Reply:
maybe a separate test that will change the value in the db to 3 and then request the version through the API to see that it changed.